src/App.jsx

```diff
@@ -1,10 +1,92 @@
 import './App.css'
+import axios from "axios";
+import {useEffect, useState} from "react";
+import PokemonCard from "./components/pokemonCard.jsx";
 
 function App() {
+    const [pokemonId, setPokemonId] = useState([])
+    const [forwardPage, setForwardPage] = useState("")
+    const [prevPage, setPrevPage] = useState("")
+    const [loading, setLoading] = useState(true)
+    const [error, setError] = useState(false)
+
+    useEffect(() => {
+        const abortController = new AbortController();
+        async function fetchPokemons () {
+            setLoading(true);
+            setError(false);
+            try {
+                const response = await axios.get("https://pokeapi.co/api/v2/pokemon/",
+                    {signal:abortController.signal,
+                })
+                setPokemonId(response.data.results)
+                setForwardPage(response.data.next)
+                setPrevPage(response.data.previous)
+            } catch (e) {
+                setError(true)
+                console.error(e.message)
+            }
+            setLoading(false)
+        }
+
+        void fetchPokemons();
+
+        // return () => {
+        //     abortController.abort()
+        // } Hij gebruikt deze abortcontroller te vroeg wanneer je dit aanzet.. Krijg je bij mount direct een cancel. How?
```

Comment:
Bij elke rerender wordt deze functie uitgevoerd, dus in principe is het niet erg dat deze te vroeg af gaat